scripts/spack/specs.json

```diff
@@ -22,7 +22,7 @@
       "clang@14.0.6+devtools+profiling" ],
 
     "blueos_3_ppc64le_ib_p9":
-    [ "clang@10.0.1+devtools+cuda+raja+umpire+profiling cuda_arch=70" ],
+    [ "clang@10.0.1~tribol+devtools+cuda+raja+umpire+profiling cuda_arch=70" ],
```

Comment:
I haven't looked at the blueos tribol cuda build in a while.  Is it not working when building with serac?  You might need to append `+cuda+raja+umpire` to the tribol spec.

Reply:
It wasn't working a very long time ago and we never went back and tried it. We should probably revisit that.

Reply:
would it hurt to always add `+raja+umpire` and only add `+cuda` to tribol if it's part of serac's spec?

```
    depends_on("tribol+raja+umpire", when="+tribol")
    depends_on("tribol+cuda", when="+tribol+cuda")
```

Reply:
if we do `tribol+raja+umpire` we may also need `~minbuild`, to prevent raja to build without openmp

Reply:
`+raja+umpire` should be fine; we currently test Tribol with serac's TPLs which have raja and umpire built.

Reply:
i decided to not include those variants for now. we can try and look into adding them once we attempt to build on lassen again in a separate pr.